test/test_statement_iterator.js

```diff
@@ -0,0 +1,61 @@
+exports.test = function(SQL, assert, done) {
```

Comment:
Taking a done callback makes the test asynchronous. Never calling `done` prevents the failing test to be reported. This is probably why you didn't notice the tests were failing.

 ```suggestion
exports.test = function(SQL, assert) {
```

Reply:
OK, I will fix it.